stable/grafana/values.yaml

```diff
@@ -143,3 +143,35 @@ grafana.ini:
     mode: console
   grafana_net:
     url: https://grafana.net
```

Comment:
Can you also include some lines in the grafana.ini section that shows how to turn on ldap? like: 
```text
grafana.ini:
  paths:
    data: /var/lib/grafana/data
    logs: /var/log/grafana
    plugins: /var/lib/grafana/plugins
  analytics:
    check_for_updates: true
  log:
    mode: console
  grafana_net:
    url: https://grafana.net
  # auth.ldap:
    # enabled = true
    # config_file = /etc/grafana/ldap.toml
```